api/db/migrations/20231218090719_add_minimumReproducibilityRateLowerLevel_to_complementaryCertifications.js

```diff
@@ -0,0 +1,26 @@
+const TABLE_NAME = 'complementary-certifications';
+const COLUMN_NAME = 'minimumReproducibilityRateLowerLevel';
+
+const up = async function (knex) {
+  await knex.schema.table(TABLE_NAME, function (table) {
+    table.decimal(COLUMN_NAME, 5, 2);
+  });
+
+  await knex(TABLE_NAME)
+    .update({
+      minimumReproducibilityRateLowerLevel: knex.ref('minimumReproducibilityRate'),
+    })
+    .where({ key: 'CLEA' });
+
+  await knex(TABLE_NAME)
+    .update({ [COLUMN_NAME]: 60 })
+    .where({ minimumReproducibilityRateLowerLevel: null });
```

Comment:
```suggestion
    .whereNull('minimumReproducibilityRateLowerLevel');
```
on pourrait aussi mettre 60 par defaut

Reply:
@aceol On est sûr de vouloir mettre 60 par défaut ? Pour l'instant on n'a pas le champs côté front pour seter la valeur. Du coup toutes les nouvelles certif comp auront 60, c'est ok ?